hack/prowimagebuilder/main.go

```diff
@@ -44,11 +46,10 @@ const (
 	defaultProwImageListFile = "prow/.prow-images.yaml"
 
 	defaultWorkersCount = 10
-
-	defaultRetry = 3
+	defaultRetry        = 3
 
 	// noOpKoDocerRepo is used when images are not pushed
-	noOpKoDocerRepo = "do.not/matter/at/all"
+	noOpKoDocerRepo = "ko.local"
```

Comment:
FWIW it is useful even when not pushing to have a predictable and reasonable name so tarballs can be `docker load` ed for development testing (or `kind load image-archive`).

Reply:
I typically use the location I _would_ have pushed to.

Reply:
For some reason ko failed to build multi-arch images into tarballs, so I chose this as alternative, hopefully no one needs to `docker load` tarball for development